Code/Framework/AzCore/AzCore/Console/ConsoleFunctor.cpp

```diff
@@ -172,4 +172,14 @@ namespace AZ
     {
         return GetValueResult::NotImplemented;
     }
+
+#if !defined(_RELEASE) && defined(CARBONATED)
+    void ConsoleFunctorBase::PerformCommand(CVarFixedString inString)
+    {
+        if (m_console)
```

Comment:
Can it be that cvars being assigned without console? Please add "else", then set a breakpoint to check. Maybe two platforms PC & iOS.